specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-10-15-preview/EventGrid.json

```diff
@@ -0,0 +1,8286 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group within the user's subscription.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "domainName",
+            "in": "path",
+            "description": "Name of the domain.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Domain"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 Bad Request.\n\n * 500 Internal Server Error."
+          }
+        },
+        "x-ms-examples": {
+          "Domains_Get": {
+            "$ref": "./examples/Domains_Get.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Domains"
+        ],
+        "summary": "Create or update a domain.",
+        "description": "Asynchronously creates or updates a new domain with the specified parameters.",
+        "operationId": "Domains_CreateOrUpdate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group within the user's subscription.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "domainName",
+            "in": "path",
+            "description": "Name of the domain.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "domainInfo",
+            "in": "body",
+            "description": "Domain information.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Domain"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/Domain"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 Bad Request.\n\n * 500 Internal Server Error."
+          }
+        },
+        "x-ms-examples": {
+          "Domains_CreateOrUpdate": {
+            "$ref": "./examples/Domains_CreateOrUpdate.json"
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
do you need to define the following?
        "x-ms-long-running-operation-options": {
          "final-state-via": "location"
        }

Reply:
We are not familiar with this property and/or what it is used for. Please note that domain is an existing resource and this definition is not new (even though it shows as new in this iteration). We will not change existing behavior/definition as this is PR as this is outside the scope of the change and we need to ensure we understand the behavior correctly to apply to all resources.. Hence, Won't fix here.